requirements.txt

```
@@ -1,7 +1,7 @@
 codecov
-pytest
+pytest==4.6.11
```

```suggestion
pytest<=4
```
Just in case they ever add a new 4.X release with bug fixes.

---

So unfortunately when running pip install I get this error:
**pytest-cov 2.10.1 requires pytest>=4.6, but you'll have pytest 4.0.0 which is incompatible.**
Should have run it locally before committing. @adamjstewart what's your preferred solution here?

---

Updated to pytest~=4.6. I'm new to python but as I understand it this should get us anything >=4.6 but keep us within major version 4. Build passing.

---

Ah, good catch. According to https://www.python.org/dev/peps/pep-0440/#inclusive-ordered-comparison:

> As with version matching, the release segment is zero padded as necessary to ensure the release segments are compared with the same length.

That explains the behavior you were seeing.